keel-core/src/main/kotlin/com/netflix/spinnaker/keel/events/ResourceHealthEvent.kt

```diff
@@ -0,0 +1,10 @@
+package com.netflix.spinnaker.keel.events
+
+/**
+ * An event emitted when we calculate resource health
+ */
+data class ResourceHealthEvent(
+  val resourceId: String,
+  val application: String,
+  val healthy: Boolean = true
```

Comment:
Is it worth adding a `comment` or `details` field (probably only useful for the unhealthy case)?

Reply:
Oh, never mind, these won't be persisted in history.